st2common/st2common/util/crypto.py

```diff
@@ -80,7 +80,8 @@
 
 DEFAULT_AES_KEY_SIZE = 256
 
-assert DEFAULT_AES_KEY_SIZE >= MINIMUM_AES_KEY_SIZE
+if not DEFAULT_AES_KEY_SIZE >= MINIMUM_AES_KEY_SIZE:
+    raise ValueError('Verify the key size :"%s".' % (DEFAULT_AES_KEY_SIZE))
```

Comment:
Needs better comparison, and an error message that indicates that the AES key size is smaller than the minimum key size.